### DISCUSSION TITLE
4 relè a contatto pulito (o a 24V) via MQTT

### DISCUSSION THREAD

**Discussion by riccionese** [indomus/forum#537] [Hardware & Firmware]
Ciao a tutti, sto valutando un progetto per il quale dovrei gestire 4 carichi (valvole) pilotate a DC 24v, possibilmente da montare su un binario DIN e che possa pilotare con MQTT
Per questo motivo sto cercando un "aggeggio" che possa avere contatti puliti o che sia pilotabile direttamente a DC 24V

Stavo guardando: 
- lo Shelly 4 Pro (o Pro Plus), che è MQTT,  ma non ha contatto pulito e non mi pare che non sia pilotabile a 24 V
- il Sonoff 4CH Pro R2 che ha il contatto pulito e sarebbe pure pilotabile direttamente a 24v ma non è MQTT a meno di passarlo a Tasmota, ma ho guardato le guide è ... non ho voglia di mettermi con saldatori ecc.
- l'unica cosa che ho trovato sarebbero 4 Shelly 1 ... ottimi (ne ho già altri per altri scopi) ma se riuscissi a trovare un dispositivo unico preferirei.

Qualcuno di voi ha suggerimenti ?
ciao e grazie

**Comment by KC-inDomus** [comment]
Sonoff 4ch pro. non hai bisogno di saldatori, ti basta mettere a contatto 4 poli collegati al ttl/usb. credi che è più lunga spiegarla che farla.

**Reply by riccionese** [reply]
ok. diciamo che il fatto che non abbia la funzione DIY (chissà perché?) come ad esempio il Mini mi aveva intimorito.
Il materiale necessario ce l'ho (FTDI e cavi Dupont)
Intanto ne prendo uno poi ci farò qualche prova nelle prossime ferie 
:-)

ciao e grazie

**Reply by riccionese** [reply]
Wait wait wait !!
vedo che è uscito il R3 e dice che è DIY ???
Qualcuno ha già provato o ha guide aggiornate a questo R3 ?

**Reply by riccionese** [reply]
ho trovato questo video 
https://www.youtube.com/watch?v=1K1vTNzsTxI 
che tratta esplicitamente gli R3 (Pro e non-Pro) ma no riesco a "non" far flashare il 5 led.
:-(
sul sito tasmota, per  l'R3, dicono di fare un ponticello 
vediamo ... 

nessuno che ha già fatto questo cinema su un R3 ?

**Reply by KC-inDomus** [reply]
in che senso a non far flashare il 5?

**Reply by riccionese** [reply]
Nel video spiega che dando corrente con il primo switch premuto per i soliti 3 secondi, entra in modalità programmazione e la condizione per avere conferma è che il quinto led sulla scheda non lampeggi.
il device ha i 4 led uno per switch ed un quinto che indica lo stato della connessione ( o qualche cosa del genere) 
In realtà, almeno con il mio 4ch pro r3 quel led lampeggia BLU con il FW originale in attesa della connessione mentre se entrati in programmazione, invece di "non lampeggiare" come indicato nel video, lampeggia ROSSO.
Ed in questa modalità funziona e sono riuscito a Tasmotizzare

**Reply by riccionese** [reply]
ok. rileggendo ho capito l'ambiguità di avere scritto "flashare" intendendo letteralmente lampeggiare e non per "scrivere il FW"
